cmd/backup-manager/app/export/export.go

```diff
@@ -101,8 +102,9 @@ func (bo *Options) backupDataToRemote(source, bucketURI string, opts []string) e
 
 	// the backup was a success
 	// remove .tmp extension
-	args = backupUtil.ConstructArgs(constants.RcloneConfigArg, opts, "moveto", tmpDestBucket, destBucket)
+	args = backupUtil.ConstructArgs(constants.RcloneConfigArg, opts, "moveto", tmpDestBucket, destBucket, true)
 	output, err = exec.Command("rclone", args...).CombinedOutput()
+	klog.Infof("rclone move data from %s data to %s. rclone log: %s", tmpDestBucket, destBucket, output)
```

Comment:
Move this after L110 and add cluster info.

Reply:
fixed